independent-publisher-connectors/snowflake/apiDefinition.swagger.json

```diff
@@ -0,0 +1,745 @@
+                      "example": 20
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Asynchronous execution in progress. Use provided query id to perform query monitoring and management.",
+            "x-ms-summary": "Message",
+            "schema": {
+              "type": "object",
+              "additionalProperties": true
+            }
+          }
+        },
+        "summary": "Check the Status and Get Results",
+        "description": "Check the Status of the Execution of a Statement and Get the Results",
+        "operationId": "GetResults",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "Instance",
+            "in": "header",
+            "type": "string",
+            "required": true,
+            "description": "The uri of your Snowflake instance (without https://)",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "default": "application/json",
+            "type": "string",
+            "required": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "default": "application/json",
+            "type": "string",
+            "required": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "statementHandle",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "requestId",
+            "in": "query",
+            "type": "string",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "partition",
+            "in": "query",
+            "type": "integer"
+          }
+        ]
+      }
+    },
+    "/{statementHandle}/cancel": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "x-ms-summary": "Message",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string",
+                  "description": "code"
+                },
+                "sqlState": {
+                  "type": "string",
+                  "description": "sqlState"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                },
+                "statementHandle": {
+                  "type": "string",
+                  "description": "statementHandle"
+                },
+                "statementStatusUrl": {
+                  "type": "string",
+                  "description": "statementStatusUrl"
+                }
+              }
+            }
+          },
+          "default": {

```

Comment:
@rekodus Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**